### PR TITLE
Feat/z searchbar recursive tree view

### PR DESCRIPTION
**Pull request by saldomik**
# Feat - ZSearchbar - Add tree item to autocomplete results

## Priority

- [ ] 1 - Highest
- [ ] 2 - High
- [x] 3 - Medium
- [ ] 4 - Low
- [ ] 5 - Lowest
- [ ] 6 - Not a Priority

## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] Feature (non-breaking change which adds functionality)
- [ ] Component (add a Component as approved by Design System)
- [ ] Docs (add documentation)
- [ ] Chore (changes that adds small enhancement)
- [ ] Breaking (fix or feature that would cause existing functionality to not work as expected)

## Design
https://www.figma.com/design/w51ZwDwLNEt0yamar2sRRQ/Nuovi-Componenti-e-Pattern?node-id=10085-500&t=6Fn6CidVWOsa3IHD-0

### CODE REVIEW

File: src/components/z-searchbar/styles.css [hunk 1]
```diff
@@ -78,12 +85,78 @@
...
+  height: 44px;
+  content: "";
+  transform: translateX(-100%);
+}
+
+.results z-list-element > .list-element:hover,
+.results z-list-element > .list-element:hover::after {
+  background-color: var(--color-surface02);
+
+  /*   background-color: yellow; */
```
Comment (Alessandro-Zuddas) [inline src/components/z-searchbar/styles.css:158]: So che ti piacerebbe lasciarlo giallo :100:

**Review by Alessandro-Zuddas — APPROVED** [review]

File: src/components/z-searchbar/index.tsx [hunk 2]
```diff
@@ -395,6 +392,13 @@ export class ZSearchbar {
           </span>
           {item?.tag && <z-tag icon={item.tag.icon}>{!this.isMobile ? item.tag.text : ""}</z-tag>}
         </div>
+        {item.children && item.children.length > 0 ? (
+          <z-list>
+            <div style={{paddingLeft: "24px"}}>
```
Comment (francescochiapella) [inline src/components/z-searchbar/index.tsx:397]: userei le var dei token, e magari sposterei in file css

File: src/components/z-searchbar/styles.css (hunk 1 above)
Comment (francescochiapella) [reply src/components/z-searchbar/styles.css:158]: convertirei le misure impostate come pixel secchi a usare le variabili css del DS

**Comment by francescochiapella** [comment]
Convertirei le misure impostate secche in Pixel per utilizzare le var css del DS

File: src/components/z-searchbar/styles.css (hunk 1 above)
Comment (saldomik) [reply src/components/z-searchbar/styles.css:158]: mi piange il cuore toglierlo

**Pull request merged**